api/test/gateways/UHW/UHWSearchGateway.test.js

```diff
@@ -68,33 +68,22 @@ describe('UHWSearchGateway', () => {
     expect(db.request).toHaveBeenCalledWith(queryMatcher, expect.anything());
   });
 
-  it('returns a record if id exists', async () => {
+  it('builds a record', async () => {
     const record = { ContactNo: 1231 };
     const gateway = createGateway([record]);
 
     const records = await gateway.execute({});
 
     expect(buildSearchRecord).toHaveBeenCalledTimes(1);
     expect(records.length).toBe(1);
-    expect(records[0].id).toBe('1231');
-  });
-
-  it("doesn't return a record if the id is missing", async () => {
-    const record = {};
-    const gateway = createGateway([record]);
-
-    const records = await gateway.execute({});
-
-    expect(buildSearchRecord).toHaveBeenCalledTimes(0);
-    expect(records.length).toBe(0);
   });
 
-  it("returns empty records if the db connection experiences an error", async () => {
+  it('returns empty records if the db connection experiences an error', async () => {
```

Comment:
This should be called something like 'it returns an empty set of records if there is an error'